Dockerfile

```diff
@@ -1,23 +1,21 @@
-FROM python:3.5-slim-stretch
+FROM python:3.7-slim-stretch
 
 RUN apt-get update && apt-get install -y \
     g++ \
     gfortran \
     libblas-dev \
     liblapack-dev \
     libopenblas-dev \
-    python3-dev \
     enchant \
     build-essential
+COPY requirements.txt /ores/requirements.txt
+COPY test-requirements.txt /ores/test-requirements.txt
+RUN pip install pip --upgrade && pip install wheel && pip install nltk \
+    && pip install -r /ores/requirements.txt \
+    && pip install -r /ores/test-requirements.txt \
+    && python -m nltk.downloader stopwords
 
 COPY . /ores
 WORKDIR /ores
 
-RUN pip install pip --upgrade
```

Comment:
Each of these `RUN` commands adds an additional layer to the image and increases the overall size. We can avoid this by concatenating these commands in a single layer instead.